OracleWebCenterSites/dockerfiles/12.2.1.4/Dockerfile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
It should be 2019, 2020 as the content is being updated in 2020. Also, please remove "All rights reserved." as we no longer include that in copyright headers in 2020.